rundeckapp/grails-app/controllers/rundeck/controllers/MenuController.groovy

```diff
@@ -2915,12 +2916,26 @@ Format is a string like `2d1h4n5s` using the following characters for time units
                     && extra.futureScheduledExecutions.size() > max) {
                 extra.futureScheduledExecutions = extra.futureScheduledExecutions[0..<max]
             }
+        } else if(request.api_version >= ApiVersions.V48) {
```

Comment:
A job may be scheduled, but also have "run later" executions, I think the `else` should be removed?